src/vi/goctruyentranhvui/src/eu/kanade/tachiyomi/extension/vi/goctruyentranhvui/GocTruyenTranhVui.kt

```diff
@@ -135,4 +146,7 @@ class GocTruyenTranhVui : HttpSource() {
         SortByList(getSortByList()),
         GenreList(getGenreList()),
     )
+    companion object {
+        private const val TOKEN_KEY = "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJBbG9uZSBGb3JldmVyIiwiY29taWNJZHMiOltdLCJyb2xlSWQiOm51bGwsImdyb3VwSWQiOm51bGwsImFkbWluIjpmYWxzZSwicmFuayI6MCwicGVybWlzc2lvbiI6W10sImlkIjoiMDAwMTA4NDQyNSIsInRlYW0iOmZhbHNlLCJpYXQiOjE3NTM2OTgyOTAsImVtYWlsIjoibnVsbCJ9.HT080LGjvzfh6XAPmdDZhf5vhnzUhXI4GU8U6tzwlnXWjgMO4VdYL1_jsSFWd-s3NBGt-OAt89XnzaQ03iqDyA"
```

Comment:
Move this constant below the class, into top level and remove the companion object.

Reply:
> I actually mean this:
> 
> ```kotlin
> class GocTruyenTranhVui : HttpSource() {
> }
> 
> private const val TOKEN_KEY = "..." // <-- here
> ```

ok i changed